File: src/main/java/info/devels/jtte/gui/JTTECreativeTab.java
package info.devels.jtte.gui;

import info.devels.jtte.blocks.BlockBeacon;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.Item;


public class JTTECreativeTab extends CreativeTabs {
    public JTTECreativeTab() {
        super("jtte");
    }

    @Override
    public Item getTabIconItem() {
        return BlockBeacon.blockBeacon.getItem();
    }
}

File: src/main/kotlin/info/devels/jtte/gui/JTTECreativeTab.kt
package info.devels.jtte.gui

import info.devels.jtte.blocks.BlockBeacon
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.Item


class JTTECreativeTab : CreativeTabs("jtte") {
    override fun getTabIconItem(): Item {
        return BlockBeacon.blockBeacon.item
    }
}
